cron/internal/bq/main.go

```diff
@@ -31,6 +32,8 @@ import (
 	"github.com/ossf/scorecard/v4/cron/internal/data"
 )
 
+var configFilename = flag.String(config.ConfigFlag, config.ConfigDefault, config.ConfigUsage)
+
```

Comment:
Can define this flag in `config.go` so that multiple definitions are not required?

Reply:
Basically,

```
// config.go

var configFileFlag = ...

func ReadConfig() err {
  // Read input file from `--config`
}
```

```
// main.go

flag.Parse()
if err := config.ReadConfig(); err != nil {
  // handle error
}
```

Reply:
I think it makes the `flag.Parse()` call opaque, but it would work